build.sh

```diff
@@ -12,6 +12,7 @@ export CFLAGS="-O3 -fomit-frame-pointer -fPIC"
 if [ "$(uname)" == "Darwin" ]; then
    export CXXFLAGS="${CXXFLAGS} -arch x86_64 -arch arm64"
    export CFLAGS="${CFLAGS} -arch x86_64 -arch arm64"
+   export MACOSX_DEPLOYMENT_TARGET=10.14
```

Comment:
Looks good to me. But @RoseHirigoyen also told me that someone recommended her to actually build FROM a machine that has the right version if possible.

I personally don't have enough knowledge to really know the differences between the two.

Reply:
Or you can use bee to download the target version SDK - I didn't do it for recorder because it is a lot of setup and the `MACOSX_DEPLOYMENT_TARGET` ended up working well, but it's a thing to keep in mind if we have unexplainable dllnotfound errors in the future!